### PR TITLE
refactoring: use Promise.all

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | refactoring: use Promise.all |
| 🇨🇳 Chinese |          无须纳入ChangeLog |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at cd4f0ea</samp>

Refactored and optimized the `generate-cssinjs.ts` script to make it faster and cleaner.

### <samp>🤖 Generated by Copilot at cd4f0ea</samp>

* Improve performance and readability of generating CSS in JS files by using `Promise.all` instead of `for` loop (`[link](https://github.com/ant-design/ant-design/pull/42219/files?diff=unified&w=0#diff-f767f34c2dcf1e08fb6be20f2ed33bb0aaa2aad39c7c97c07c71ab7c7193033cL22-R43)`)

### CODE REVIEW

File: scripts/generate-cssinjs.ts [hunk 1]
```diff
@@ -19,28 +18,27 @@ export const styleFiles = globSync(
...
-    const componentName = pathArr[styleIndex - 1];
-    let useStyle: StyleFn = () => {};
-    if (file.includes('grid')) {
-      // eslint-disable-next-line no-await-in-loop
-      const { useColStyle, useRowStyle } = await import(file);
-      useStyle = (prefixCls: string) => {
-        useRowStyle(prefixCls);
-        useColStyle(prefixCls);
+  await Promise.all(
+    styleFiles.map<ReturnType<GenCssinjs>>(async (file) => {
```
Comment (MadCcc) [inline scripts/generate-cssinjs.ts:22]: 这个类型好像无关吧，不写也没关系

Comment (li-jia-nan) [reply scripts/generate-cssinjs.ts]: 删掉了

**Comment by MadCcc** [comment]
这样子的话外层就不需要 async/await 了，可以这样
```
export const genCssinjs = (...args) => Promise.all(stylefiles.map(async (item) => {...}))

**Comment by li-jia-nan** [comment]
> > 这样子的话外层就不需要 async/await 了，可以这样
> > ```js
> > export const genCssinjs = (...args) => Promise.all(stylefiles.map(async (item) => { /* ... */ }))
> > ```
> 
> 好，我晚点改一下

done~

**Review by MadCcc — APPROVED** [review]

**Pull request merged**